modules/core/PrimDisc.cpp

```diff
@@ -7,11 +7,13 @@ namespace rt {
 	{
 		float dist = (m_origin - ray.org).dot(m_normal) / ray.dir.dot(m_normal);
 		if (dist < Epsilon || isinf(dist) || dist > ray.t) return false;
-		if (static_cast<float>(norm(ray.org + ray.dir * dist - m_origin)) > m_radius) return false;
+		if (static_cast<float>(norm(ray.org + ray.dir * dist - m_origin)) <= m_radius) return false;
```

Comment:
I think this is an error, should be `> m_radius`

Reply:
Right, I was debugging and I forgot to put it back the way it should be